scripts/ci/run-step.sh

```diff
@@ -26,6 +26,10 @@ case $i in
     TIMEOUT="${i#*=}"
     shift # past argument=value
     ;;
+    --retry=*)
+    RETRY="${i#*=}"
+    shift
+	;;
```

Comment:
we apparently use spaces for indentation in this file, please keep it consistent.